src/main/java/net/pms/util/AudioUtils.java

```diff
@@ -101,10 +121,254 @@ public static String getLPCMChannelMappingForMencoder(DLNAMediaAudio audioTrack)
+						new String(signature, StandardCharsets.US_ASCII)
+					);
+				}
+				return false;
+			}
+			media.setContainer(FormatConfiguration.RA);
+			short version = buffer.getShort();
+			int reportedHeaderSize = 0;
+			int reportedDataSize = 0;
+			if (version == 3) {
+				audio.setCodecA("RealAudio 14.4");
+				audio.getAudioProperties().setNumberOfChannels(1);
+				audio.getAudioProperties().setSampleFrequency(8000);
+				short headerSize = buffer.getShort();
+
+				buffer = ByteBuffer.allocate(headerSize);
+				channel.read(buffer);
+				buffer.flip();
+				buffer.position(8);
+				int bytesPerMinute = buffer.getShort() & 0xFFFF;
+				reportedDataSize = buffer.getInt();
+				byte b = buffer.get();
+				if (b != 0) {
+					byte[] title = new byte[b & 0xFF];
+					buffer.get(title);
+					String titleString = new String(title, StandardCharsets.US_ASCII);
+					audio.setSongname(titleString);
+					audio.setAudioTrackTitleFromMetadata(titleString);
+				}
+				b = buffer.get();
+				if (b != 0) {
+					byte[] artist = new byte[b & 0xFF];
+					buffer.get(artist);
+					audio.setArtist(new String(artist, StandardCharsets.US_ASCII));
+				}
+				audio.setBitRate(bytesPerMinute * 8 / 60);
+				media.setBitrate(bytesPerMinute * 8 / 60);
+			} else if (version == 4 || version == 5) {
+				buffer = ByteBuffer.allocate(14);
+				channel.read(buffer);
+				buffer.flip();
+				buffer.get(signature);
+				if (!".ra4".equals(new String(signature, StandardCharsets.US_ASCII))) {
+					LOGGER.debug("Invalid RealAudio 2.0 signature \"{}\"", new String(signature, StandardCharsets.US_ASCII));
+					return false;
+				}
+				reportedDataSize = buffer.getInt();
+				buffer.getShort(); //skip version repeated
+				reportedHeaderSize = buffer.getInt();
+
+				buffer = ByteBuffer.allocate(reportedHeaderSize);
+				channel.read(buffer);
+				buffer.flip();
+				buffer.getShort(); // skip codec flavor
+				buffer.getInt(); // skip coded frame size
+				buffer.getInt(); // skip unknown
+				long bytesPerMinute = buffer.getInt() & 0xFFFFFFFFL;
+				buffer.getInt(); // skip unknown
+				buffer.getShort(); // skip sub packet
+				buffer.getShort(); // skip frame size
+				buffer.getShort(); // skip sub packet size
+				buffer.getShort(); // skip unknown
+				if (version == 5) {
+					buffer.position(buffer.position() + 6); // skip unknown
+				}
+				short sampleRate = buffer.getShort();
+				buffer.getShort(); // skip unknown
+				short sampleSize = buffer.getShort();
+				short nrChannels = buffer.getShort();
+				byte[] fourCC;
+				if (version == 4) {
+					buffer.position(buffer.get() + buffer.position()); // skip interleaver id
+					fourCC = new byte[buffer.get()];
+					buffer.get(fourCC);
+				} else {
+					buffer.getFloat(); // skip deinterlace id
+					fourCC = new byte[4];
+					buffer.get(fourCC);
+				}
+				String fourCCString = new String(fourCC, StandardCharsets.US_ASCII).toLowerCase(Locale.ROOT);
```

Comment:
Wrong, the reason why it's displayed in lower case is this: a4f837958f623c178d0132c3d24509e54f87c371

What I set to lower case is the "fourCC" code. The reason for doing that is to be sure that I get a hit in the following switch statement independent of the case used in the fourCC code in the file.